syncd/NotificationProcessor.cpp

```diff
@@ -463,6 +463,28 @@ void NotificationProcessor::process_on_queue_deadlock_event(
     sendNotification(SAI_SWITCH_NOTIFICATION_NAME_QUEUE_PFC_DEADLOCK, s);
 }
 
+
+void NotificationProcessor::process_on_port_host_tx_ready_change(
+        _In_ sai_object_id_t switch_id,
+        _In_ sai_object_id_t port_id,
+        _In_ sai_port_host_tx_ready_status_t *host_tx_ready_status)
+{
+    SWSS_LOG_ENTER();
+
+    SWSS_LOG_DEBUG("Port ID before translating from RID to VID is 0x%lx", port_id);
+    sai_object_id_t port_vid = m_translator->translateRidToVid(port_id, SAI_NULL_OBJECT_ID);
+    SWSS_LOG_DEBUG("Port ID after translating from RID to VID is 0x%lx", port_vid);
```

Comment:
you can use sai_serialize_object_id(port_id).c_str() to have string which will also add oid: prefix

Reply:
Done